django/utils/timezone.py

```diff
@@ -74,7 +76,7 @@ def get_default_timezone():
 
     This is the time zone defined by settings.TIME_ZONE.
     """
-    return pytz.timezone(settings.TIME_ZONE)
+    return utc if settings.TIME_ZONE.upper() == 'UTC' else pytz.timezone(settings.TIME_ZONE)
```

Comment:
Perhaps it might worth to add function for this as we're using `pytz.timezone()` in other places.

Reply:
Perhaps a comment should be added explaining why the special case for utc is useful (I guess it's for performance?), and also why values like 'uTC' are accepted.